mono/utils/mono-proclib.c

```diff
@@ -944,3 +948,29 @@ mono_cpu_usage (MonoCpuUsageState *prev)
 
 	return cpu_usage;
 }
+
+size_t
+mono_get_total_physical_memory (void)
```

Comment:
there's already almost identical code in https://github.com/mono/mono/blob/f5836956c6c7d4564469ae446cd44d04d01646d4/mono/metadata/mono-perfcounters.c#L414. Extract to a common helper?